pkg/plugin/plugin.go

```diff
@@ -147,3 +164,14 @@ func secretVersion(ctx context.Context, secretVersionName string) (string, error
 
 	return string(resp.GetPayload().GetData()), nil
 }
+
+func standardInternalErr(justificationValue string) error {
+	return status.Errorf(codes.Internal, "internal error, unable to validate jira issue %q", justificationValue)
+}
```

Comment:
Isn't the code already indicating it's internal error?
```suggestion
func internalErr(justificationValue string) error {
	return status.Errorf(codes.Internal, "unable to validate jira issue %q", justificationValue)
}
```